packages/e2e-tests/specs/editor/various/writing-flow.test.js

```diff
@@ -512,9 +512,9 @@ describe( 'Writing Flow', () => {
 		const y = paragraphRect.y + paragraphRect.height + 1;
 
 		await page.mouse.move( x, y );
-		await page.waitForSelector(
-			'.block-editor-block-list__insertion-point-inserter'
-		);
+		// await page.waitForSelector(
```

Comment:
Why is it commented out? It doesn't seem like it should have any impact on toolbars.

Reply:
Ah, I forgot it there. That testing was failing while I was trying to fix the ordering issue. I was playing with the test code to see if there was some problem in the test implementation. Just reverted this.